cl/assets/templates/includes/date_picker.html

```diff
@@ -0,0 +1,23 @@
+{% if DEBUG %}
+  <script defer type="application/javascript"
+          src="{{ STATIC_URL }}js/bootstrap-datepicker.js"></script>
+{% else %}
+  <script defer type="application/javascript"
+          src="{{ STATIC_URL }}js/bootstrap-datepicker.min.js"></script>
+{% endif %}
+<script type="text/javascript">
+  $(document).ready(function () {
+    $('.datepicker').datepicker({
+      autoclose: true,
+      assumeNearbyYear: true,
+      forceParse: false,
+      maxViewMode: "decades",
+      startDate: "01/01/1600",
```

Comment:
I would disable future dates to the date picker.  I believe the setting is 

```endDate: '+0d',```

Reply:
I went back and forth on this one, but I think you're right that it'll help folks more than hinder. I'll tweak it.